javers-core/src/main/java/org/javers/core/metamodel/clazz/ShallowReferenceDefinition.java

```diff
@@ -0,0 +1,14 @@
+package org.javers.core.metamodel.clazz;
+
+import org.javers.core.metamodel.type.EntityType;
+
+/**
+ * Recipe for {@link EntityType}.
+ *
+ * @author akrystian
+ */
+public class ShallowReferenceDefinition extends EntityDefinition {
```

Comment:
mam trochę kłopot z tym dziedziczeniem,
czy to znaczy że ShallowReference **jest** rodzajem Encji?